lib/libspl/include/os/windows/string.h

```diff
@@ -23,7 +23,7 @@
 #ifndef _LIBSPL_WINDOWS_STRING_H
 #define	_LIBSPL_WINDOWS_STRING_H
 
-#pragma message ("this we want")
```

Comment:
Heh honestly, this is debug from when I was trying to figure out the include problem and hierarchy. It should be removed - but it can come later